File: 99.育成中コンテンツ/広告/アドフリくん/adfurikunSDK_Android_moviereward_3.20.0/SampleJava/app/src/main/java/sample/adfurikunsdk/jp/samplejava/admob/SimpleAdMobBanner.java
package sample.adfurikunsdk.jp.samplejava.admob;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.FrameLayout;
import android.widget.Toast;

import sample.adfurikunsdk.jp.samplejava.R;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.google.android.gms.ads.AdListener;
import com.google.android.gms.ads.AdSize;
import com.google.android.gms.ads.LoadAdError;
import com.google.android.gms.ads.MobileAds;
import com.google.android.gms.ads.admanager.AdManagerAdRequest;
import com.google.android.gms.ads.admanager.AdManagerAdView;

public class SimpleAdMobBanner extends Activity {

    private static final String AD_UNIT_ID = "";

    private FrameLayout mAdContainer;
    private AdManagerAdView mAdView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_admob_banner);

        mAdContainer = findViewById(R.id.ad_container);

        findViewById(R.id.load).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if (mAdView != null) {
                    mAdView.loadAd(new AdManagerAdRequest.Builder().build());
                }
            }
        });

        // AdMob初期化
        MobileAds.initialize(this);

        // AdMobのバナー表示用ビューを生成
        mAdView = new AdManagerAdView(this);
        mAdView.setAdSizes(AdSize.BANNER);
        mAdView.setAdUnitId(AD_UNIT_ID);
        mAdView.setAdListener(mAdListener);
        mAdContainer.addView(mAdView);
    }

    /**
     * 広告読み込み・表示用リスナー
     */
    private AdListener mAdListener = new AdListener() {
        @Override
        public void onAdLoaded() {
            super.onAdLoaded();
            // 広告読み込み完了時に呼ばれる。
            Toast.makeText(SimpleAdMobBanner.this, "読み込み成功", Toast.LENGTH_SHORT).show();
        }

        @Override
        public void onAdFailedToLoad(@NonNull LoadAdError loadAdError) {
            super.onAdFailedToLoad(loadAdError);
            // 広告読み込み失敗時に呼ばれる。
            Toast.makeText(SimpleAdMobBanner.this, "読み込み失敗", Toast.LENGTH_SHORT).show();
        }

        @Override
        public void onAdImpression() {
            super.onAdImpression();
            // 広告表示時に呼ばれる。
            Toast.makeText(SimpleAdMobBanner.this, "広告表示", Toast.LENGTH_SHORT).show();
        }

        @Override
        public void onAdOpened() {
            super.onAdOpened();
            // 広告をクリックした時に呼ばれる。
            Toast.makeText(SimpleAdMobBanner.this, "広告クリック", Toast.LENGTH_SHORT).show();
        }
    };

    @Override
    protected void onResume() {
        super.onResume();
        // 広告を再開
        if (mAdView != null) {
            mAdView.resume();
        }
    }

    @Override
    protected void onPause() {
        if (mAdView != null) {
            mAdView.pause();
        }
        // 広告を一時停止
        super.onPause();
    }

    @Override
    protected void onDestroy() {
        // 広告を破棄
        if (mAdView != null) {
            mAdView.destroy();
        }
        super.onDestroy();
    }
}

File: 広告/アドフリくん/adfurikunSDK_Android_moviereward_3.20.0/SampleKotlin/app/src/main/java/jp/glossom/sample/samplekotlin/admob/SimpleAdMobBanner.kt
package jp.glossom.sample.samplekotlin.admob

import android.app.Activity
import android.os.Bundle
import android.widget.Button
import android.widget.FrameLayout
import android.widget.Toast
import com.google.android.gms.ads.AdListener
import com.google.android.gms.ads.AdSize
import com.google.android.gms.ads.LoadAdError
import com.google.android.gms.ads.MobileAds
import com.google.android.gms.ads.admanager.AdManagerAdRequest
import com.google.android.gms.ads.admanager.AdManagerAdView
import jp.glossom.sample.samplekotlin.R

class SimpleAdMobBanner : Activity() {

    private lateinit var mAdContainer: FrameLayout

    private val AD_UNIT_ID = ""
    private var mAdView: AdManagerAdView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_admob_banner)

        mAdContainer = findViewById(R.id.ad_container)

        findViewById<Button>(R.id.load).setOnClickListener {
            val adRequest = AdManagerAdRequest.Builder().build()
            mAdView?.loadAd(adRequest)
        }

        // AdMob初期化
        MobileAds.initialize(this)

        // AdMobのバナー表示用ビューを生成
        mAdView = AdManagerAdView(this).apply {
            setAdSizes(AdSize.BANNER)
            adUnitId = AD_UNIT_ID
            adListener = mAdListener
        }
        mAdContainer.addView(mAdView)
    }

    /**
     * 広告読み込み・表示用リスナー
     */
    private val mAdListener = object : AdListener() {
        override fun onAdLoaded() {
            // 広告読み込み完了時に呼ばれる。
            Toast.makeText(this@SimpleAdMobBanner, "読み込み成功", Toast.LENGTH_SHORT).show()
        }

        override fun onAdFailedToLoad(adError: LoadAdError) {
            // 広告読み込み失敗時に呼ばれる。
            Toast.makeText(this@SimpleAdMobBanner, "読み込み失敗", Toast.LENGTH_SHORT).show()
        }

        override fun onAdImpression() {
            super.onAdImpression()
            // 広告表示時に呼ばれる
            Toast.makeText(this@SimpleAdMobBanner, "広告表示", Toast.LENGTH_SHORT).show()
        }

        override fun onAdClicked() {
            super.onAdClicked()
            // 広告をクリックした時に呼ばれる。
            Toast.makeText(this@SimpleAdMobBanner, "広告クリック", Toast.LENGTH_SHORT).show()
        }
    }

    override fun onResume() {
        super.onResume()
        // 広告を再開
        mAdView?.resume()
    }

    override fun onPause() {
        mAdView?.pause()
        // 広告を一時停止
        super.onPause()
    }

    override fun onDestroy() {
        // 広告を破棄
        mAdView?.destroy()
        super.onDestroy()
    }
}